zou/app/blueprints/previews/__init__.py

```diff
@@ -99,6 +100,10 @@
         "/actions/preview-files/<preview_file_id>/set-main-preview",
         SetMainPreviewResource,
     ),
+    (
+        "/data/assets/<instance_id>.png/asset-asset-instances",
```

Comment:
This route name is not relevant at all, we work on preview files not on assets...
It should be something like : "/data/preview-files/<preview_file_id>/extract-frame"

Reply:
We need the frame number in the route: "/data/preview-files/<preview_file_id>/extract-frame/23".
 